src/constants.ts

```diff
@@ -34,3 +34,7 @@ export const ApiSpecificationOptions = {
     localFile: UiStrings.ApiSpecificationOptionLocalFile,
     activeEditor: UiStrings.ApiSpecificationOptionActiveEditor,
 };
+export const DefinitionFormat = {
```

Comment:
Better to put into https://github.com/microsoft/vscode-azureapicenter/blob/dev/src/azure/ApiCenter/contracts.ts

Reply:
updated